ludwig/schema/model_config.py

```diff
@@ -527,19 +527,21 @@ def _set_hyperopt_defaults(self):
 
         max_t = scheduler.get("max_t")
         time_attr = scheduler.get("time_attr")
-        epochs = self.trainer.to_dict().get("epochs", None)
+        epochs_key = "epochs" if self.model_type == MODEL_ECD else "num_boost_round"
```

Comment:
For GBMs, "epochs" is determined by `num_boost_round // boosting_rounds_per_checkpoint`, isn't it?

Reply:
Or are we simplifying the logic here by only using the total number of boosting rounds ?

Reply:
@jeffkinnison I believe there's no exact equivalent to epochs for GBM. What would be the desired logic here?